changelog/unreleased/bug-fixes/1563.md

```diff
@@ -0,0 +1,4 @@
+The shutdown logic contained a bug that would make the node fail to terminate
+in case a plugin actor is registered at said node.
+
+A race condition in the shutdown logic that caused an assertion was fixed.
```

Comment:
Every paragraph is a changelog entry of its own, I doubt this is what you wanted to achieve here.

Reply:
These are 2 separate Bugs that are both fixed in this PR.